specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/examples/ServerCreate.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "serverName": "pgtestsvc4",
+    "resourceGroupName": "TestGroup",
+    "api-version": "2020-02-14-privatepreview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff",
+    "parameters": {
+      "location": "westus",
+      "properties": {
+        "administratorLogin": "cloudsa",
+        "administratorLoginPassword": "password",
+        "serverEdition": "GeneralPurpose",
+        "vCores": 4,
+        "storageQuotaInMb": 524288,
+        "version": "12",
+        "standbyCount": 0,
+        "backupRetentionDays": 7,
+        "vnetInjArgs": {
+          "delegatedSubnetName": "test-subnet",
+          "delegatedVnetID": "6ed504a3-24b8-4ba8-ac1f-7a1780195437",
+          "delegatedVnetName": "test-vnet"
```

Comment:
We are introducing one more property here delegatedVnetResourceGroup, https://msdata.visualstudio.com/Database%20Systems/_git/orcasql-breadth/pullrequest/389410